database/mongodb/mongodb.go

```diff
@@ -23,6 +25,10 @@ func init() {
 
 var DefaultMigrationsCollection = "schema_migrations"
 
+const DefaultLockingCollection = "lock"
+const DefaultLockingKey = "key"
```

Comment:
minor, maybe this should be called "locking key" or something?

for (even more) clarity on what this does :smile: